apps/admin-gui/src/app/shared/components/users-list/users-list.component.html

```diff
@@ -74,7 +74,8 @@
         <tr *matHeaderRowDef="displayedColumns" mat-header-row></tr>
         <tr
           *matRowDef="let user; columns: displayedColumns;"
-          [perunWebAppsForceRouterLink]="inDialog ? null : ['/admin/users', user.id]"
+          [perunWebAppsForceRouterLink]="(inDialog || disableRouting) ? null : ['/admin/users', user.id]"
```

Comment:
maybe this could be reworked in the users-list so that we dont have two indicators for the indication if the routing in the list is enabled. Rework it so there is only disableRouting.